examples/pytorch/gpt/utils/huggingface_bloom_convert.py

```diff
@@ -209,9 +209,8 @@ def handle_exceptions(model_config: PretrainedConfig,
     return param
 
 
-def convert_and_save_parameter(config: PretrainedConfig,
-                               name: str,
-                               param: torch.nn.Parameter,
+def convert_and_save_parameter(param_name: str,
```

Comment:
You change the API, but don't update the line 333.

Reply:
I changed that in the following commit. But maybe we can just remove if else statement and use star_async. 1 process can still work under this condition.

Reply:
I would assume everyone even the laptop now should have 4 cores, so 1 core condition not common